api/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "phone_directory",
+  "version": "1.0.0",
+  "description": "a simple api to find all the phone number",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "nodemon src/index.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/sathyarjunab/node-phone-directory.git"
+  },
+  "author": "sathyarjun007",
```

Comment:
change to sathyarjunab this is your git user name

Reply:
done